src/_locales/en/messages.json

```diff
@@ -132,10 +132,10 @@
     "message": "Whether generated passwords should include symbols by default"
   },
   "passff_prefs_default_include_symbols_label": {
-    "message": "Include symbols by default"
+    "message": "Include at least one special character in the password by default"
   },
   "passff_prefs_preferred_new_method_tooltip": {
-    "message": "Whether new passwords default to being generated or being inserted"
+    "message": "Whether new passwords default to be automatically generated or to be inserted by user"
```

Comment:
The verb "to default to sth." is used with the "-ing" form of the verb (source: my experience + Google). Can you give a source for the contrary?

Reply:
My bad, you are right.